src/dbnode/storage/types.go

```diff
@@ -1394,6 +1400,21 @@ type AggregateTilesOptions struct {
 	InsOptions instrument.Options
 }
 
+// Aggregator is the interface for AggregateTiles.
+type Aggregator interface {
```

Comment:
Note: I'm not completely sure of the naming here. I'm afraid this might get confused with `m3aggregator`.
Would putting it into a `tile` package (to make it a `tile.Aggregaor`) be a better option? I'm open to other suggestions as well.

Reply:
Maybe just `TileAggregator`? There's also the `AggregateQuery` query type (which handles tag completion and was named after the [ES concept](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations.html)) to add further confusion here

Reply:
Renamed as suggested 👍 .